src/archive/src/main.rs

```diff
@@ -421,54 +422,49 @@ fn http_request(req: HttpRequest) -> HttpResponse {
 fn encode_metrics(w: &mut MetricsEncoder<Vec<u8>>) -> std::io::Result<()> {
     with_config(|config| {
         w.encode_gauge(
-            "ii_archive_last_upgrade_timestamp",
-            config.last_upgrade_timestamp as f64,
+            "ii_archive_last_upgrade_timestamp_seconds",
+            Duration::from_nanos(config.last_upgrade_timestamp).as_secs_f64(),
             "Timestamp of the last upgrade of this canister.",
         )
     })?;
     with_log(|log| {
         w.encode_gauge(
-            "ii_archive_log_entries_count",
+            "ii_archive_entries_count{source=\"log\"}",
             log.len() as f64,
             "Number of log entries stored in this canister.",
         )?;
         w.encode_gauge(
-            "ii_archive_log_entries_size",
-            log.log_size_bytes() as f64,
-            "Total size of all logged entries in bytes, not counting data structure overhead.",
+            "ii_archive_log_bytes_total{memory=\"data\"}",
```

Comment:
`_total` is a suffix that denotes counters (monotonically increasing metrics that keep track of the number of occurrences of an event). This is (and should continue to be) a gauge, it should not end in `_total`.

Same everywhere else.